### PR TITLE
Added merchant_faq_es.md file

### PR DESCRIPTION
**Pull request by lh1008**
Hey everyone, 

Finished 'Monero Merchant FAQs' Spanish translation.

https://www.monerooutreach.org/merchants/monero-merchant-faqs.html

Pinging @quagmir3 @rodolfo912 @BlackLotus64 @arawako for review.

:)

### CODE REVIEW

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (dilubit) [inline monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:3]: Instead of "nos gustaría saber que otras podrían haber" , I'd change to "nos gustaría saber qué otras podrías tener"

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (dilubit) [inline monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:7]: Instead of  "Los comerciantes tienen la manera de hablar con los clientes de cómo se protege su privacidad", I'd change to "Los comerciantes tienen la posibilidad de hablar con los clientes de cómo proteger su privacidad"

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (dilubit) [inline monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:15]: Instead of "Monero continúa siendo cada vez más eficiente, la tendencia en términos de comisiones es tener comisiones más bajas que las de Bitcoin.", I'd change to:  "Monero continúa siendo cada vez más eficiente y  la tendencia es tener comisiones cada vez mas bajas. "

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (dilubit) [inline monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:27]: Comisión **por** Switch

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (dilubit) [inline monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:28]: I'd remove the USD symbol. It's duplicate with $

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (dilubit) [inline monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:32]: I'd change this:  "transacciones débito o crédito"  to this: "transacciones de débito o crédito"

and I'd say: 3023 instead of 3.023

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (dilubit) [inline monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:48]: I'd change from:  "en los negocios que existen en las fronteras " to this:  "en los comercios fronterizos"

Comment (dilubit) [inline monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:48]: Instead imponible, I'd use imputado.

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (dilubit) [inline monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:55]: Add:  "Cómo seleccionar el procesador de pagos correcto para tu negocio"

**Review by dilubit — COMMENTED** [review]
@lh1008 Review done!

**Comment by dilubit** [comment]
@lh1008  I did the review

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (lh1008) [reply monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:11]: Done

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (lh1008) [reply monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:15]: Changed :).

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (lh1008) [reply monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:27]: You're right. Done.

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (lh1008) [reply monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:28]: Well, the main reason I left it is because it's a reference to the USD, in other countries the symbol **'$'** represents money but it doesn't make a reference to a currency. E.g, the 'peso' doesn't have a symbol like the EUR (€) or the GBP (£), and its monetary representation comes accompanied with the **'$'** sign.

Comment (lh1008) [reply monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:28]: So, removing the '$' symbol could confuse readers' conversion rate.

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (lh1008) [reply monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:32]: Thanks :). Done.

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (lh1008) [reply monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:48]: Yes, this reads much better. Changed :). Done.

Comment (lh1008) [reply monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:48]: I left "imponible" because of its exact definition: 'something that can be taxed or tributed'. 
The word "imputable" means: 'something than can be linked to', in this case, yes a tax but it could change the meaning of a product that can be taxed.
Let me know what you think.

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (lh1008) [reply monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:55]: Where?

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (dilubit) [reply monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:28]: I know, but you're using the symbol two times: $ and USD. That's why I would keep with one of these.

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (dilubit) [reply monero-outreach-docs/translations/es/merchants/merchant_faq_es.md]: you 're right

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (dilubit) [reply monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:55]: After the link. Here: (https://www.monerooutreach.org/merchants/monero-payment-processor-guide.html) "Cómo seleccionar el procesador de pagos correcto para tu negocio"

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (lh1008) [reply monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:28]: Okay, I understand. I'll remove it.

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (lh1008) [reply monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:55]: We might leave the translation title that hasn't been translated yet :P. 

"Procesadores de pagos de Monero"

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (lh1008) [reply monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:28]: You're right, I'll removed the '$' symbol.

File: monero-outreach-docs/translations/es/merchants/merchant_faq_es.md
Comment (dilubit) [reply monero-outreach-docs/translations/es/merchants/merchant_faq_es.md:55]: ok!

**Comment by dilubit** [comment]
It's ok for me

**Review by dilubit — APPROVED** [review]
@lh1008  ok!

**Review by lh1008 — COMMENTED** [review]
@erciccione these changes were approved. It says it needs an extra approval, I guess from the maintainers. :)

**Comment by erciccione** [comment]
@lh1008 It needs an approval from one of the maintainers, me or you. As we set it.

**Review by erciccione — APPROVED** [review]

**Pull request merged**
